source/js/foundation/template-js-handler/article-hero-video-handler.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Adding play/pause functionality to the optional Article Page hero video section.
+ */
+
+ const articleHeroVideoHandler = () => {
+    let articlePageHero = document.querySelector(".article-hero");
+  
+    if (!articlePageHero) {
+      return;
+    }
+  
+    let video = articlePageHero.querySelector("video.hero-video");
+    let pauseButton = articlePageHero.querySelector(".btn-pause");
+    let playButton = articlePageHero.querySelector(".btn-play");
+  
+    function toggleVideo() {
+        video.paused ? video.play() : video.pause();     
+    }
+
+
+
```

Comment:
this is a lot of whitespace, suggesting that `prettier` isn't picking it up. Let's make sure it does.